### PR TITLE
Add initial weight exporting scripts

### PR DESCRIPTION
**Pull request by ianjjohnson**
At this point, we're not updating validation metadata or the like, just including a basic scripts to take classification weights trained on ImageNet and export them to GCS.

I expect this to evolve a lot over time, but it is a starting point.

@LukeWood

### CODE REVIEW

File: shell/weights/process_weights.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+if [ "$#" -ne 4 ]; then
```
Comment (LukeWood) [inline shell/weights/process_weights.sh:1]: how hard would it be to add a JSON file based on the TensorBoard logs to `weights/densenet/classification/history.json'?

Comment (ianjjohnson) [reply shell/weights/process_weights.sh:1]: I've added a separate python script to perform this extraction and update the JSON file. Since it's logically separate from the weights processing, I'm not referencing it from this shell script.

At this point in time, it's only extracting validation accuracy. But it's a starting point from which to add other metrics such as commit author, number of epochs trained, etc.

**Review by LukeWood — APPROVED** [review]
looks good, I am glad we can start including these scripts & JSON files.

File: shell/weights/remove_top.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+FLAGS(sys.argv)
+
+if not FLAGS.weights_path.endswith(".h5"):
+    raise ValueError("Weights path must end in .h5")
+
+model = eval(
+    f"keras_cv.models.{FLAGS.model_name}(include_rescaling=True, include_top=True, num_classes=1000, weights=FLAGS.weights_path)"
+)
+
+without_top = keras.models.Model(model.input, model.layers[-3].output)
```
[the remark this answers is not in the document]
Comment (ianjjohnson) [reply shell/weights/remove_top.py:22]: Didn't see this before this got merged -- I just checked and this is always -3 except for VGG19 where it looks like it's -5. I'll submit a fix which makes this more robust whenever I run this against VGG weights.

[the remark this answers is not in the document]
Comment (LukeWood) [reply shell/weights/remove_top.py:22]: SGTM!
